### ISSUE TITLE
How does Motivation_Structure represent morpheme breaks?

### ISSUE THREAD

**Issue by fmatter** [cldf/cldf#121]
The description for [motivationStructure](https://cldf.clld.org/v1.0/terms.html#motivationStructure) says

> As an example, consider Chinese *shùpí* "bark (of a tree)" which is a compound consisting of *shù* "tree" and *pí* "skin", and whose motivation structure could be rendered as tree bark.

Assuming a `FormTable` like

| ID | Form  | Parameter_ID | Motivation_Structure |
|----|-------|--------------|----------------------|
| 1  | shùpí | tree_bark    | tree bark            |

...how do we know which segments correspond to 'tree' and which to 'bark'? Or is that not the purpose, is it just about encoding that these meanings are in the form?

**Comment by xrotwang** [comment]
@LinguList could you have a look?

**Comment by LinguList** [comment]
Ah, sorry, did not see this right away.

**Comment by LinguList** [comment]
@fmatter, `Motivation_Structure` assumes that you can split your form into as many parts as the motivation structure.

**Comment by fmatter** [comment]
> @fmatter, Motivation_Structure assumes that you can split your form into as many parts as the motivation structure.

I got that, but where should it be split?

> typically, we only define it for cases where we have segmented data in IPA (in our normalized https://clts.clld.org/).

Is this referring to the space-separated `Segments` column? Or something else?

**Comment by LinguList** [comment]
Here is a blog post, where the major idea is discussed as it is used in the EDICTOR tool, from which we create CLDF in turn: https://calc.hypotheses.org/2735

So the answer is `shùpí` would not work, it should be `shù-pí`, and you should make it clear that `-` is your morpheme separator for the form. Or even better `ʂ u ⁵¹+ p i ³⁵` in the Segments column.

I'd also say that `Morpheme_Structure` pertains to lexical operations. So in a IGT example, you make your concordance, and then you would derive a lexicon of your data, where you would add this annotation along with phonetic transcriptions using orthography profiles. This was at least our workflow in the original paper introducing pyigt.

**Comment by fmatter** [comment]
> Here is a blog post, where the major idea is discussed as it is used in the EDICTOR tool, from which we create CLDF in turn: https://calc.hypotheses.org/2735
> 
> So the answer is `shùpí` would not work, it should be `shù-pí`, and you should make it clear that `-` is your morpheme separator for the form. Or even better `ʂ u ⁵¹+ p i ³⁵` in the Segments column.

Right, ok. How do you currently handle secondary separators (` `)?
